### ISSUE TITLE
PyQt4 is deprecated

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Chameleon#4]
PyQt4 is deprecated. When replacing with PyQt5 I get:

```
AttributeError: module 'PyQt5.QtGui' has no attribute 'QWizard'
```

I wanted to use Chameleon today but it looks like I cannot run it anymore.

**Comment by illwieckz** [comment]
@Viech, would you know how to port it to PyQt5?

**Comment by illwieckz** [comment]
It looks like I have a [work-in-progress](https://github.com/DaemonEngine/Chameleon/files/6328097/chameleon-pyqt5-wip.patch.txt)unfinished commit from 2019. I believe it does I stopped my effort because I did not know how to port signals.

**Comment by Viech** [comment]
> @Viech, would you know how to port Chameleon to PyQt5?

No idea, sorry. Haven't built a GUI application since my last commit here.

**Comment by illwieckz** [comment]
OK, I'm looking for help because otherwise, this tool is dead, which is a shame because it's a god-send (or a @Viech-send, that's the same, right? 😁).

The more recent currently supported Ubuntu distro able to run Chameleon today is Bionic, which is from April of 2018, while maintained, that's a three years old distro, that's bad. I also noticed an Ubuntu Eoan (Octobre of 2019) can run it but this version is discontinued and repositories were taken down (there is still an usable archive somewhere though).

We really need to fix that. 😱

**Comment by IngarKCT** [comment]
Needs to be ported to PyQt6, Qt5 is done for.

**Comment by illwieckz** [comment]
It looks like I got it ported to Qt5 to begin with, well, there is some deprecation warning and the shader cache does not work, but the application is fully working.

**Comment by illwieckz** [comment]
Fixed by #5, remaining thing to do reported in #6.
